kiwi/bootloader/config/grub2.py

```diff
@@ -750,6 +750,12 @@ def _setup_secure_boot_efi_image(self, lookup_path, uuid=None, mbrid=None):
                     os.sep.join([self.efi_boot_path, grub_image.binaryname])
                 ]
             )
+            mok_manager = os.sep.join(
+                [os.path.dirname(shim_image), 'MokManager.efi']
```

Comment:
Both Fedora and Ubuntu use `mm<EFIARCH>.efi` instead of `MokManager.efi`.

Reply:
Uhm, ok yet another translation table...

Reply:
thanks

Reply:
done now

Reply:
@Conan-Kudo this should do the trick. if suse switches the name to use what mm<EFIARCH>.efi the match table will continue to apply